run_pacta.R

```diff
@@ -0,0 +1,382 @@
+# load packages----
+# library(dotenv)
+library(dplyr)
+library(pacta.multi.loanbook.analysis)
+library(r2dii.analysis)
+library(r2dii.data)
+library(r2dii.plot)
+
+
+# source helpers----
+source("expected_columns.R")
+source("plots.R")
+
+# load config----
+config <- config::get()
+
+input_path_scenario <- config$directories$dir_scenario
```

Comment:
nit: I have renamed some of the objects created from the confg in the other scripts, basically stripping the prefix `input_`. s anything that is a full path to a file is `path_* <- x/y.ext` and anything that is a directory is `dir_* <- x/y`. Not technically necessary to merge this, but maybe a good idea to use one style across the scripts.

Reply:
💯 agree, but I think I'll wait to do that in a separate repo-wide review/alignment

Reply:
- see #54